sdk/typescript/src/client/wasm-loader.ts

```diff
@@ -65,17 +65,12 @@ export async function generate_secp_cg_keypair_from_seed(
 	return wasm.generate_secp_cg_keypair_from_seed(seed);
 }
 
-export async function create_dkg_centralized_output(
+export async function create_dkg_centralized_output_v1(
```

Comment:
add a `v2` function too, and we should call it from the `JS` accordingly, so add a TODO + issue link here